dataprofiler/tests/profilers/profiler_options/test_categorical_options.py

```diff
@@ -32,6 +83,8 @@ def test_set(self):
         params_to_check = [
             dict(prop="is_enabled", value_list=[False, True]),
             dict(prop="top_k_categories", value_list=[None, 3]),
+            dict(prop="max_sample_size_to_check_stop_condition", value_list=[None, 20]),
```

Comment:
and to @JGSweets comment too here -- shouldn't we test this with a `float` to validate its caught

Reply:
this isn't checking validation helper